src/test/ui/repeat_count.stderr

```diff
@@ -33,22 +33,12 @@ error[E0308]: mismatched types
    |
 LL |     let f = [0; -4_isize];
    |                 ^^^^^^^^ expected `usize`, found `isize`
-   |
-help: you can convert an `isize` to `usize` and panic if the converted value wouldn't fit
```

Comment:
Here too... but this time it's arrays, so there's a conflict in the heuristic. Can we refactor and keep this somehow though?